pkg/driver/availability/error.go

```diff
@@ -0,0 +1,25 @@
+package availability
+
+import (
+	"errors"
+)
+
+type Error interface {
+	Error() string
```

Comment:
You'll need to add a unexported method of some bogus name to this otherwise this will be implemented by any error

Reply:
Alternatively you can use a no interface approach for errors like this. Look at the must rebuild error in RDK if you're interested

Reply:
Done. Used the no interface approach. Works nicely.